include/linkdefs.h

```diff
@@ -29,6 +29,7 @@ enum RPNCommand {
 	RPN_DIV		= 0x03,
 	RPN_MOD		= 0x04,
 	RPN_UNSUB	= 0x05,
+	RPN_EXP		= 0x06,
```

Comment:
Since this changes object file semantics, the object file version must be bumped.

Reply:
Done.